.github/ISSUE_TEMPLATE/0-bug-report.yaml

```diff
@@ -0,0 +1,311 @@
+        - 9.3.1
+        - 9.3.0
+        - 9.2.10
+        - 9.2.8
+        - 9.2.7
+        - 9.2.6
+        - 9.2.5
+        - 9.2.4
+        - 9.2.3
+        - 9.2.2
+        - 9.2.1
+        - 9.2.0
+        - 9.1.x
+        - 9.0.x
+        - 8.5.x
+        - 8.4.x
+        - 8.3.x
+        - 8.2.x
+        - 8.1.x
+        - 8.0.x
+        - 7.x.x or older
+    validations:
+      required: true  
+  - type: markdown
+    attributes:
+      value: |
+       
+       # Is the bug inside a Dashboard Panel?
+
+  - type: textarea
+    id: get-help
+    attributes:
+      label: |
+        
+        ## Optional Questions:
+        
+        ### Is the bug inside a Dashboard Panel?
+
+      description: |
+        If the bug appears inside a [dashboard panel](https://grafana.com/docs/grafana/latest/panels-visualizations/#panels-and-visualizations), please use the ["Get-Help" feature](https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/). Select **copy to clipboard** and paste the data below.
+        No need for backticks--this text will get formatted as a code-block.
+
+        What's a [dashboard panel](https://grafana.com/docs/grafana/latest/panels-visualizations/#panels-and-visualizations)?
+      placeholder: Copy "get-help" data here
+      value: Copy the panel's ["get-help" data](https://grafana.com/docs/grafana/latest/troubleshooting/send-panel-to-grafana-support/) here  
+  - type: markdown
+    attributes:
+      value: |
+        #
+        #
+        # Optional Questions:
+        #
+  - type: markdown
+    attributes:
+      value: |
+       #
+       # Did you check your grafana server logs?
+  - type: textarea
+    id: logs
+    attributes:
+      label: |
+        
+        ## Optional Questions:
+
+        ### Any relevant logs?
+
+      description: |
+        Please copy and paste any relevant log output. You might want to increase your Grafana log's verbosity to 'Debug'
+        No need for backticks--this text will get formatted as a code-block.
+        [How do I find my grafana server logs](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#logs)?
+        [How do I increase my grafana server logs to 'debug'](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#log)?
+      render: sh
+      placeholder: Copy your grafana server logs here
+      value: Copy your grafana server logs here  
+  - type: markdown
+    attributes:
+      value: |
+        
+        # Can you share more configuration details?
+        
+  - type: dropdown
+    id: gf-deployment
+    attributes:
+      label: Grafana Platform?
+      description: |
+        **How** are you running/deploying Grafana?
+      options:
+        - I use Grafana Cloud
+        - Docker
+        - Kubernetes
+        - A package manager (APT, YUM, BREW, etc.)
+        - A downloaded binary
+        - Other
+        - I don't know
+    validations:
+      required: false
+  - type: dropdown
```

Comment:
Without versions, this information is only semi-useful. Can we make the OS field a free text form?

Reply:
love it. absolutely. will move towards required freetext fields